src/components/analyses/details/Drawer.js

```diff
@@ -70,14 +70,6 @@ const useStyles = makeStyles((theme) => ({
         maxWidth: "100%",
     },
 
-    tabIndicator: {
-        backgroundColor: theme.palette.secondary.main,
-        color: theme.palette.secondary.contrastText,
-    },
-    tabSelected: {
-        color: theme.palette.primary.contrastText,
-        backgroundColor: theme.palette.primary.main,
-    },
```

Comment:
I noticed these styles were duplicated for tabs in a few places, so I moved them to a common file with `DETabs`, `DETab`, and `DETabPanel`